contrib/osx/keybindings.el

```diff
@@ -7,14 +7,17 @@
 
   ;; Keybindings
   (global-set-key (kbd "s-q") 'save-buffers-kill-terminal)
-  (global-set-key (kbd "s-v") 'yank)
-  (global-set-key (kbd "s-c") 'kill-ring-save)
+  (global-set-key (kbd "s-v") 'evil-paste-after)
+  (global-set-key (kbd "s-c") 'evil-yank)
+  (global-set-key (kbd "s-a") 'mark-whole-buffer)
   (global-set-key (kbd "s-x") 'kill-region)
   (global-set-key (kbd "s-w") 'kill-this-buffer)
-  (global-set-key (kbd "s-z") 'undo-tree-undo)
```

Comment:
This is wrong, spacemacs uses a special undo thingy that has fancy tree-ness. Try pressing `SPC a u`. The undo and redo bindings should not be changed.